### DISCUSSION TITLE
Can't logon / connect to Soulseek server

### DISCUSSION THREAD

**Discussion by shanghaiultra** [slskd/slskd#859] [Q&A] [answered]
Keep getting the error in the log "Not connecting to the Soulseek server; username and/or password invalid. Specify valid credentials and manually connect, or update config and restart"

I am fairly sure the logon details are correct as I've verified them on another fully functional installation of Slsk for Linux Mint. I've also tried a test login and pw which in theory should be no problem since this is how you make new accounts on slsk.

So does the message mean the issue is Slskd can't reach the Soulseek server? My slsk running on linux mint has no issues and is running on a vm in the same machine. My docker installation can ping the internet so I am stumped.

Grateful for any suggestions. Wondering if I have overlooked something obvious since this doesn't seem to be a problem experienced by anyone else.

**Comment by shanghaiultra** [comment, marked as answer]
Solved - obvious indeed. Forgot to remove # from the yaml file for the soulseek password and login sections. Doh. 
Thanks to the project owner for this. Have been trying to find a way to make soulseek run headlessly for years.

**Reply by gpuma** [reply]
I also struggled with this. Looking at a fully commented out file to me suggests that the file is somehow being uncommented by the program during runtime. To me this seems unintuitive.

**Comment by jpdillingham** [comment]
Unfortunately it's very easy to miss things when editing YAML, you're not the first person to do this (myself included)

I've created issue #860 to hopefully make this easier.

**Comment by shanghaiultra** [comment]
Great work. Thanks once again!
